docs/docs/how-to-contribute.md

```diff
@@ -58,7 +58,7 @@ The usual contributing steps are:
 - Setup up repo and install dependencies: `yarn run bootstrap`
 - Make sure tests are passing for you: `yarn test`
 - Create a topic branch: `git checkout -b topics/new-feature-name`
-- Run `npm run watch` from the root of the repo to watch for changes to packages' source code and compile these changes on-the-fly as you work. Note that the watch command can be resource intensive. To limit it to the packages you're working on, add a scope flag, like `npm run watch -- --scope={gatsby,gatsby-cli}`. To watch just one package, run `npm run watch -- --scope=gatsby`.
+- Run `yarn run watch` from the root of the repo to watch for changes to packages' source code and compile these changes on-the-fly as you work. Note that the watch command can be resource intensive. To limit it to the packages you're working on, add a scope flag, like `yarn run watch -- --scope={gatsby,gatsby-cli}`. To watch just one package, run `yarn run watch -- --scope=gatsby`.
```

Comment:
Why did you change the npm commands?

Reply:
It seemed inconsistent, everywhere else in this doc `yarn` was used instead of `npm`

I just assumed it was overlooked, happy to drop that commit as well

Reply:
> ensure you have the latest version of yarn installed (>= 1.0.2)

Ahh, I guess it's fine then 👍

Reply:
The docs _do_ use npm everywhere, except in these contributing docs where people are asked to use yarn instead.